lib/mastermind/main_runner.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Mastermind
+  class MainRunner
+
+    class << self
+      def run
+        new.run
+      end
+    end
+
+    def initialize
+
+    end
+
+    def run
+      puts 'And I raAaAan! I ran so far awaAaAay!'
+    end
+  end
+end
```

Comment:
@corlassion I would recommend re-writing your script as a ruby class like this one.